umm-spec-lib/test/cmr/umm_spec/test/umm_g/sanitizer.clj

```diff
@@ -17,4 +17,5 @@
       (update :platform-refs (fn [x] (when (seq x) (distinct x))))
       (util/update-in-each
        [:platform-refs]
-       #(util/update-in-each % [:instrument-refs] update :operation-modes sanitize-operation-modes))))
+       #(util/update-in-each % [:instrument-refs] update :operation-modes sanitize-operation-modes))
+      (util/update-in-each [:orbit-calculated-spatial-domains] #(assoc % :orbit-number nil))))
```

Comment:
Isn't orbit number a valid field in both the umm-lib granule model and UMM-G? I'm guessing there's a rule that it can include either an orbit-number or both a start and stop orbit number.

It seems like we're not testing both scenarios.

Reply:
That's what I'm gonna point out too.  umm-lib allows orbit-number and start, stop orbit numbers to co-exist, but UMM-G doesn't.  So, we may have the invalid data in the system. Also, UMM-G requires start/stop orbit numbers to be integer, before my merge, umm-lib allowed them to be double, so there might be invalid data in the system as well.

Reply:
Updated generator to generate the record according to UMM-G constraint. I don't want to mask the data problem during conversion. The bad data needs to be fixed through granule update. I would rather deal with it later when the need becomes apparent, but I did add in a simple double to int conversion to deal with the orbit number type change that Siwei made.